### DISCUSSION TITLE
how to use Tree-sitter Textobject Based Navigation with german Keyboard layout

### DISCUSSION THREAD

**Discussion by e-Forest** [helix-editor/helix#3107] [Q&A] [answered]
Hello together,

to be able to use "Tree-sitter Textobject Based Navigation" one need to type "[" or "]".

With english keyboard-layout it is no problem. If one type "[" or "]" a popup apear with further options.

On my german keyboard layout the symbols "[" and "]" are only typeable in combination with "AltGr" but when I try this combinations nothing happens.

Can anybody help out here?

Thank you in advance.

<img width="449" alt="image" src="https://user-images.githubusercontent.com/50156472/179734696-87e5d0f5-9738-422b-9a51-21edd572017e.png">

**Comment by sudormrfbin** [comment, marked as answer]
Are you using Windows? There's an open issue about AltGr not working on it: https://github.com/helix-editor/helix/issues/2914.

**Reply by e-Forest** [reply]
Yes I am using Windows.
Thanks that answers my question 👍
